content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -21,16 +21,21 @@ DVC makes it easy to track [metrics](/doc/command-reference/metrics), update
 <admon type="tip">
 
 All of the above can be combined into [experiments] to run and compare many
-iterations of your ML project.
+iterations of your machine learning project.
 
 [experiments]: /doc/start/experiments
 
 </admon>
 
 ## Collecting metrics
 
-First, let's see the mechanism to capture values for these ML attributes. Add
-and run a final evaluation stage to our [earlier pipeline]:
+Collecting and visualizing metrics is crucial for machine learning
+experimentation. However, DVC's flexible mechanism to deal with them generalizes
```

Comment:
`However, DVC's flexible mechanism to deal with them generalizes ` -> something is wrong here with language. And I don't this the sentence is needed. `beyond model training use cases` - let's not complicate (raises questions, etc).

`and you can use it with any process that writes metrics into one or more files upon execution.` - it's too technical and too early

Overall, same here. It should be 1-2 sentences (w/o losing any meaning or information)

Reply:
> something is wrong here with language

I beg to differ :) The sentence seems to be correct 🤔 Please explain

> And I don't this the sentence is needed. beyond model training use cases - let's not complicate (raises questions, etc).

Respectfully disagree. I usually don't bother arguing about such things but let me explain the motivation - in the original version we go to demonstrate using `-M` to designate a metric files without explaining at all what it even means  - what are "metrics" for DVC.
I don't mind changing the explanation, but it is again about throwing readers of during a get-started tutorial - you need to explain what's happening. the user doesn't have the context, they don't know what metrics mean at this point at all

Reply:
> what are "metrics" for DVC.

yes, I see the reasoning in your update overall. This specific sentence doesn't feel like give any new information for "what are "metrics" for DVC".

> I don't mind changing the explanation, but it is again about throwing readers off during a get-started tutorial - you need to explain what's happening.

yes, feels too verbose and repetitive. I'm not saying it's not needed at all. I just hope (and we should try to do this at the very top and in a 1-2-3 sentences) or modify some existing paragraphs to accommodate some clarifications if possible.